tests/test_api.py

```diff
@@ -175,7 +175,7 @@ def test_retrieve(client): #TODO Need to write supporting end point and library
     try:
         # populate db with blob and metadata
         conn, cur = db_connect('testing')
-        cur.execute('INSERT INTO blob VALUES(%s, %s,%s)',('hash1','f/hash1','saved'))
+        cur.execute('INSERT INTO blob VALUES(%s, %s)',('hash1','f/hash1'))
```

Comment:
are you gonna rewrite these with sqlalchemy?

Reply:
done